integration/fuse/src/main/java/alluxio/fuse/auth/AuthPolicy.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.auth;
+
+import alluxio.AlluxioURI;
+
+/**
+ * Fuse Auth Policy Interface.
+ */
+public interface AuthPolicy {
+  /**
+   * set use and group.
```

Comment:
Sets user and group if needed.
<add an extra empty line>
@param ...

Reply:
done